### PR TITLE
docs: example of accessing outputSettings from style hooks

### PR DESCRIPTION
**Pull request by elliot-nelson**
### SUMMARY

Add an explanation and some examples of how to access `outputSettings` from style hooks, and why you might do so.

### DETAILS

Like it says on the tin.  This might actually be an _unintended_ feature, but I think it's useful!

### CODE REVIEW

File: source/docs/style-hooks.md [hunk 1]
```diff
@@ -147,3 +147,47 @@ Here is a list of sywac styles out in the wild.
 > Tip:
 >
 > Know of a sywac style package, or have created your own? Submit a pull request for this documentation and add it to the list!
+
+## Using output settings in your style
+
+Your style hooks also have access to the user's [.outputSettings](/docs/sync-config.html#outputSettings-settings) configuration. Each property (such as `.indent`, `.lineSep`, etc.) is available as a property on `this`.
```
Comment (nexdrew) [inline source/docs/style-hooks.md:153]: Think I'd prefer the link here to point to just `/docs/sync-config.html#outputSettings` (the explicit anchor) instead, but we can clean that up later.

**Review by nexdrew — APPROVED** [review]
Looks good, thanks!

This was definitely not an intended feature, and I'm a little hesitant to recommend people rely on it, but I don't think there's any harm. Would be nice to add a test or two to sywac to make sure we don't break this in the future.

**Comment by nexdrew** [comment]
This change is now live. Thanks again!

**Comment by elliot-nelson** [comment]
Your comment is fair - I came up with this one while working on https://github.com/elliot-nelson/sywac-style-chunky (a style I've been tinkering with that I'll eventually add to the Styles list in the docs).

I think being able to insert `${this.lineSep}` is better than `\n`, but there might be a better API for it (perhaps an explicit last argument, like `fn(str, settings)`, so it doesn't break arrow functions).

**Comment by nexdrew** [comment]
> but there might be a better API for it (perhaps an explicit last argument, like `fn(str, settings)`, so it doesn't break arrow functions)

I like that idea.

**Pull request merged**